data/hai/hai reveal 1 intro.txt

```diff
@@ -11,47 +11,835 @@
+
+event "hr: sandsnap awoken"
+	planet Sandsnap
+		government "Hai Merchant"
+		attributes hai station
+		landscape land/station35
+		music ambient/machinery
+		description `This station was recently re-powered by a private military contractor to serve as a resupply and repair base that Hai warships can be stationed at to respond more rapidly to pirates entering the system. There has been a constant presence here since it was set up shortly after the pirate raids on Allhome.`
+		spaceport `The operating model for the spaceport is evidently based on the military base at Everhope, but Sandsnap's layout makes restricting portions of the promenade unfeasible. While this leaves the dual-level promenade fully open to civilian access, security restrictions have been put in place at all access points to the higher levels.`
+		`	The floor has had new texturing laid down to prevent slipping, but nearly every other surface appears to be coated in a fine, sticky, metallic powder. Frequently touched surfaces have been polished clean, but otherwise any accidental contact with the powder results in fingerprints being left on any non-metallic surface until it is thoroughly wiped off.`
+		bribe 0.8
+		security 0.8
+
+
+mission "Hai Reveal Concern Period: Sandsnap Failed"
+	landing
+	invisible
+	source "Sandsnap"
+	to offer
+		has "Hai Reveal Concern Period: Securing Sandsnap: Failed"
+	on offer
+		log `Failed to escort the convoy safely to Sandsnap. The station will not be reactivated any time soon.`
+		conversation
+			`The Stationkeeper is distraught and palms you off to her subordinate, who confirms that there is no payment he can give you.`
+			`	He explains that, without the initial convoy arriving safely, there are now loans that were available contingent on success and that they simply don't have the credits. It will take some time to raise the capital for a second attempt, if there is one.`
+				decline
+
+
+
+mission "Hai Leaks Response 2"
+	name "Go to Hai-home"
+	description "Check in with the diplomats on <destination>."
+	landing
+	source
+		government "Hai"
+		attributes "spaceport"
+		not planet "Hai-home"
+	destination "Hai-home"
+	to offer
+		has "event: hr: concern period"
+		or
+			has "Hai Reveal Concern Period: Raid on Allhome (away): declined"
+			has "Hai Reveal Concern Period: Raid on Allhome (local): declined"
+		or
+			has "Hai Reveal Concern Period: Reactivating Sandsnap: declined"
+			has "Hai Reveal Concern Period: Securing Sandsnap: done"
+			has "Hai Reveal Concern Period: Securing Sandsnap: failed"
+	on offer
+		conversation
+			`A brief announcement in the regular news feeds advises that the Hai government and the human diplomats on Hai-home are close to taking next actionable steps towards resolving the current situation. As you listen, you realize it's now been over two months since you last saw them.`
```

Comment:
More like over two years. Granted, I waited two years between the end of vanilla Hai Reveal and this PR, but considering you have to go out of your way to continue, I can see it taking a while before you get back to this.

Reply:
Again, it's an optional side quest, it's possible to progress to this point in just over 2 months.

Reply:
Yeah, but it also is possible to take longer. However, as it looks like you want it to be an optional side quest, there could be an event after a certain amount of time passes and you don’t do the missions, that forces you to move on, similarly to the hai exploration timer.

Reply:
There is, the raid on allhome stuff happens regardless of whether you do the sidequest and it proceeds from there.